features/git-sync/current_branch/feature_branch/tags.feature

```diff
@@ -0,0 +1,22 @@
+Feature: git sync: syncing a feature branch pulls tags
+
+  As a developer using Git tags for release management
+  I want that tags are pulled automatically for me whenever I sync
+  So that my local workspace has the same tags that exist on the remote
+
+
+  Background:
+    Given I have a feature branch named "feature"
+    And I have the following tags
+      | NAME       | LOCATION |
+      | local-tag  | local    |
+      | remote-tag | remote   |
+    And I am on the "feature" branch
+    And I run `git sync`
+
+
+  Scenario: result
+    Then I now have the following tags
+      | NAME       | LOCATION         |
+      | local-tag  | local            |
+      | remote-tag | local and remote |
```

Comment:
So tags are automatically pulled if they are attached to commits you normally pull (ie: a commit on a branch). But if the tag is not on a commit that gets pulled down (ie: a commit that is not on a branch), then it would not get the tag. That case can be handled in a separate issue but wanted to point out that this feature test is not sufficient